ansible.html.markdown

```diff
@@ -490,7 +490,7 @@ You might like to know, that you can build your inventory dynamically.
 
 (For Ansible) inventory is just a JSON with proper structure - if you can deliver that to ansible - anything is possible.
 
-You do not need to invent the wheel - there are plenty ready to use inventory script for most popular Cloud provicers and a lot of in-house popular usecaseses.
+You do not need to invent the wheel - there are plenty ready to use inventory script for most popular Cloud provicers and a lot of in-house popular usecases.
```

Comment:
You do not need to reinvent the wheel - there are plenty of ready to use inventory scripts for most popular Cloud providers and a lot of in-house popular use cases.